### ISSUE TITLE
Fail to get token in the header

### ISSUE THREAD

**Issue by desdevol** [websanova/vue-auth#603]
Version: ^3.3.8

I could not get the token to save in the local storage nor the session storage. I put a `console.log()` in the `bearer.js` and found out that `token   = headers.Authorization || headers.authorization;` returns undefined. I am not sure which part did I do wrong. I am using Vue CLI for this setup.

Dependencies:
```json
"dependencies": {
        "@websanova/vue-auth": "^3.3.8",
        "core-js": "^3.6.5",
        "vue": "^2.6.11",
        "vue-axios": "^2.1.4",
        "vue-router": "^3.2.0",
        "vuex": "^3.4.0",
        "axios": "^0.20.0"
    },
    "devDependencies": {
        "@vue/cli-plugin-babel": "~4.4.0",
        "@vue/cli-plugin-eslint": "~4.4.0",
        "@vue/cli-plugin-router": "~4.4.0",
        "@vue/cli-plugin-unit-jest": "~4.4.0",
        "@vue/cli-plugin-vuex": "~4.4.0",
        "@vue/cli-service": "~4.4.0",
        "@vue/eslint-config-prettier": "^6.0.0",
        "@vue/test-utils": "^1.0.3",
        "babel-eslint": "^10.1.0",
        "eslint": "^6.7.2",
        "eslint-plugin-prettier": "^3.1.3",
        "eslint-plugin-vue": "^6.2.2",
        "node-sass": "^4.12.0",
        "prettier": "^1.19.1",
        "sass-loader": "^8.0.2",
        "vue-template-compiler": "^2.6.11"
    }
```

My `login` function: 
```js
login: function() {
            let vm = this;
            this.$auth
                .login({
                    data: {
                        email: vm.loginForm.email,
                        password: vm.loginForm.password,
                    },
                    staySignedIn: true,
                })
                .then(
                    () => {
                        // handle redirection
                        vm.$router.push({ name: "profile" });
                    },
                    error => {
                        // Failed to login
                        alert(error);
                    },
                );
        },
```
This is my config file:
```js
import bearer from "@websanova/vue-auth/drivers/auth/bearer";
import axios from "@websanova/vue-auth/drivers/http/axios.1.x.js";
import router from "@websanova/vue-auth/drivers/router/vue-router.2.x.js";
// Auth base configuration some of this options
// can be override in method calls
const config = {
    auth: bearer,
    http: axios,
    router: router,
    tokenDefaultKey: "auth_vue_spa",
    stores: ["storage"],
    rolesKey: "role",
    registerData: {
        url: "/auth/register",
        method: "POST",
        redirect: "/login",
    },
    loginData: {
        url: "/auth/login",
        method: "POST",
        redirect: "/",
    },
    logoutData: {
        url: "auth/logout",
        method: "POST",
        redirect: "/",
        makeRequest: true,
    },
    fetchData: { url: "user", method: "GET", enabled: true },
    refreshData: {
        url: "auth/refresh",
        method: "GET",
        enabled: true,
        interval: 30,
    },
};
export default config;

```

I am using Laravel for API and the response header does return `Authorization`.
![image](https://user-images.githubusercontent.com/42801236/97399705-a6a2b780-1928-11eb-83b8-bb618b441184.png)

**Comment by websanova** [comment]
Because your token is not in bearer format, try the basic driver ;-)

**Comment by desdevol** [comment]
That does not seemed to work as well. 
```js
response: function (res) {
        var headers = this.http.getHeaders.call(this, res),
            token   = headers.Authorization || headers.authorization;

        console.log(token)
        
        return token;
    }
```
This returns undefined for me.

**Comment by websanova** [comment]
Not sure then, nothing fancy going on there, some other issue if it can't read the headers.

you can perhaps try digging in a bit on your own `conosle.log(headers)`... etc, etc and see what's up ;-)

**Comment by desdevol** [comment]
Turns out that it is a CORS problem comes from my API. For those who is experiencing similar issue, check your API if it's exposing the `Authorization` header.

This didn't happen to me when I built my Laravel-Vue SPA in one source code folder. This time I am building a separated Vue & Laravel base. If you happened to be using Laravel as well. `fruitcake/laravel-cors`  should be handling CORS issue for you already. Go to `cors.php` in config folder and make sure the header is exposed `'exposed_headers' => ['Authorization']`.
